src/containerapp/azext_containerapp/commands.py

```diff
@@ -50,26 +50,49 @@ def load_command_table(self, _):
     with self.command_group('containerapp env dapr-component') as g:
         g.custom_command('init', 'init_dapr_components', is_preview=True)
 
-    with self.command_group('containerapp service', is_preview=True) as g:
+    with self.command_group('containerapp service', deprecate_info=self.deprecate(redirect='containerapp addon', hide=True), is_preview=True) as g:
         g.custom_command('list', 'list_all_services')
 
-    with self.command_group('containerapp service redis') as g:
+    with self.command_group('containerapp addon', is_preview=True) as g:
```

Comment:
what using `containerapp add-on`?

Reply:
sorry, i don't get your question...help explain more?

Reply:
but by the section that you commented on, the addon command group is the equivalent of the service one

Reply:
The question is should we use `az containerapp add-on` instead of using `az containerapp addon` ? @bgashirabake

Reply:
oh got it got it. in discussions with @SimonJakesch, addon was what was decided on, but i just checked the docs and add-on is what's being used. i can push the change for add-on in the morning but if release time is before then, i say we go with addon. the change is straight-forward so should be seamless if made in time